packages/core/admin/admin/src/core/apis/index.js

```diff
@@ -3,3 +3,4 @@ export { default as Components } from './Components';
 export { default as Middlewares } from './Middlewares';
 export { default as Plugin } from './Plugin';
 export { default as Reducers } from './Reducers';
+export { default as customFields } from './CustomFields';
```

Comment:
Capitalize?

Reply:
I think it's because unlike the others it's just an instance of the class, not a function that returns instances

Reply:
Can we keep it consistent?

Reply:
That would be fine for me, but this was a change [required by Alex](https://github.com/strapi/strapi/pull/13718#discussion_r920061979)